docs/reference/setup/install/targz-start.asciidoc

```diff
@@ -8,9 +8,26 @@ of the {version} {es} artifacts. If macOS Catalina displays an unidentified
 developer warning when you first run {es}, you will need to add a security
 override.
 
-To add a security override, follow the instructions in
-https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac[Open
-a Mac app from an unidentified developer].
+To add a security override, run the following command on the downloaded
```

Comment:
I think this documentation will end up here https://www.elastic.co/guide/en/elasticsearch/reference/current/starting-elasticsearch.html, correct? Perhaps the header of this section could be `.macOS Gatekeeper warnings`.

The popup will not be for an `unidentified developer` warning, the text will be
```
"jdk.app" can't be opened because Apple cannot check it for malicious software.

This software needs to be updated. Contact the developer for more information.
```

Perhaps the line:
```
If macOS Catalina displays an unidentified developer warning when you first run {es}, you will need to add a security override.
```
Could be changed to:
```
If macOS Catalina displays a dialog when you first run {es} that interrupts it, you will need to take an action to allow it to run.
```

The `xattr` commands are not adding a security override, they are removing an extended attribute that causes macOS to perform some checks on the file with a tool called Gatekeeper, perhaps the line
```
To add a security override, run the following command on the downloaded
`.tar.gz` archive or the directory to which was extracted:
```
Could be:
```
To prevent Gatekeeper checks, run the following command on the downloaded
`.tar.gz` archive or the directory to which was extracted:
```

Reply:
Thank you for the helpful feedback @Conky5. I incorporated your text suggestions with 46132325f84b70296368d8691dcb1e92b537d2fe. I also relocated the note with a5c0d98cc22f3dc9085e8c4439a753e903ba8a01.

>I think this documentation will end up here https://www.elastic.co/guide/en/elasticsearch/reference/current/starting-elasticsearch.html, correct? 

Yes. You can see the current version (which this updated) here:
https://www.elastic.co/guide/en/elasticsearch/reference/7.6/starting-elasticsearch.html#_running_elasticsearch_from_the_command_line

However, I've now relocated the note below the run instructions.

Reply:
👍